modules/ROOT/pages/notifications/all-notifications.adoc

```diff
@@ -148,7 +148,7 @@ RETURN p
 
 Description of the returned code::
 Using shortest path with an exhaustive search fallback might cause query slow down since shortest path graph algorithms might not work for this use case.
```

Comment:
```suggestion
Using shortest path with an exhaustive search fallback might result in a slower query since shortest path graph algorithms might not work for this use case.
```

or if you like to keep it as a noun
```suggestion
Using shortest path with an exhaustive search fallback might cause query slowdown since shortest path graph algorithms might not work for this use case.
```